src/btree/bt_read.c

```diff
@@ -282,6 +282,9 @@ __wt_page_in_func(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags
     if (!LF_ISSET(WT_READ_CACHE))
         WT_STAT_CONN_DATA_INCR(session, cache_pages_requested);
 
+    /* Encourage a race */
```

Comment:
Could you please expand the comment? Maybe "If configured more aggressively free stashed memory to encourage finding bugs in generation tracking code"

Reply:
Thank for the suggestion, updated in e66645c5f72d1ec0a17dee9e52fa12dc83a9a9ba